### DISCUSSION TITLE
Is there a way to escape the variable syntax, so we can filter for event properties that begin with a $ sign?

### DISCUSSION THREAD

**Discussion by tomzorz** [datalust/seq-tickets#1881] [Help] [answered]
Title basically sums it up, I can't really do `where $env = 'something'` because seq assumes it's a variable: 

![image](https://github.com/datalust/seq-tickets/assets/6363312/871fc7eb-2997-496e-9ecd-ffa07ce5024d)

**Comment by liammclennan** [comment, marked as answer]
Hi @tomzorz 

`@Properties['$env'] = 'something'`

If in doubt, you can click the ✓ next to the property and choose 'find'. That will show the correct syntax.

**Reply by tomzorz** [reply]
Ooh of course, that works :) Thank you. Not sure why I didn't think of that. (Might be worth adding as a note on the variables documentation page?)
